ClinGen/clincoded
# Connects to #1129 bug new variant curation button

Merging in existing fix to address the display of "Cannot read property 'resetSelectedOption' of undefined" on the select-variant page when moving from one variant evaluation to a new variant.

To test:

1. Click on "New Variant Curation"
2. Enter a new variant, and save it
3. Start Interpretation: "Interpretation +" on Evidence Only page
4. **Without evaluating any criteria** click "View Summary" 
5. Click "New Variant Curation" 

Normal "Search and Select Variant" page should be seen
